src/Handler/Table/Preview/PreviewTableHandler.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Table\Preview;
+
+use Google\Protobuf\Internal\GPBType;
+use Google\Protobuf\Internal\Message;
+use Google\Protobuf\Internal\RepeatedField;
+use Google\Protobuf\NullValue;
+use Google\Protobuf\Value;
+use Keboola\StorageDriver\Command\Table\PreviewTableCommand;
+use Keboola\StorageDriver\Command\Table\PreviewTableResponse;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Shared\Utils\ProtobufHelper;
+use Keboola\StorageDriver\Teradata\TeradataSessionManager;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+
+class PreviewTableHandler implements DriverCommandHandlerInterface
+{
+    public const STRING_MAX_LENGTH = 50;
```

Comment:
jak pouzivat truncated, je spravne 50 znaku nebo treba 1000 znaku?

Reply:
TODO nejspis se to ma orezat primo v query na 16384 znaku (viz ExasolExportQueryBuilder::processSelectStatement), ale je tam zaroven podminka, ze exportni format musi byt JSON -> takze to tady nejspis neni vubec potreba?

Reply:
@jirkasemmler co tohle, mel bys k tomu nejaky nazor?
delat - nedelat

Reply:
jo, zazdil jsem to, hned..

Reply:
no imho by to tady (ve query) melo byt. Prvne je to ve vsech backendech a imho to vychazi z toho, ze nechceme v JSON odpovedi posilat prilis velka data. Nenasel jsem, ze by JSON me nejaky limit by design (vetsinou to pak zalezi na implementaci de/serializeru).

Jak pises, je to z duvodu exportniho formatu. Ten tady ale nemame ale asi bychom meli mit, protoze se domnivam, ze by to mohlo byt efektivnejsi (je to asi pak potreba zmerit) udelat na urovni query nez to pak nejak prohanet phpkem. 

Prozatim bych to nechal byt, ale do dalsi faze prosim zalinkuj primo tuto diskuzi at se na to pak navaze